storagemarket/provider.go

```diff
@@ -182,10 +179,12 @@ func (p *Provider) ExecuteDeal(dp *types.DealParams, clientPeer peer.ID) (pi *ap
 		Transfer:           dp.Transfer,
 	}
 	// validate the deal proposal
-	if err := p.validateDealProposal(ds); err != nil {
-		return &api.ProviderDealRejectionInfo{
-			Reason: fmt.Sprintf("failed validation: %s", err),
-		}, nil
+	if !testMode {
```

Comment:
Can we add a signature to the deal proposal so we can avoid this testMode hack? (There's an example of how to do it in the dummy deal test)

Reply:
Hey, please see https://github.com/filecoin-project/boost/pull/193#discussion_r786684209 but FWIW, I've move this var as a field on the Provider for now.

Reply:
Ok makes sense